modules/virt-creating-vddk-image.adoc

```diff
@@ -20,16 +19,16 @@ You can convert the VMware Virtual Disk Development Kit (VDDK) into an image, an
 $ mkdir /tmp/<dir_name> && cd /tmp/<dir_name>
 ----
 
-. In a browser, navigate to link:https://www.vmware.com/support/pubs/[VMware Documentation].
-. Click *VMware SDK & API Product Documentation* -> *VMware Virtual Disk Development Kit (VDDK)*.
-. Select the latest VDDK release and click *Download SDKs*.
-. Extract the archive within your temporary directory:
+. In a browser, navigate to link:https://code.vmware.com/home[VMware code] and click *SDKs*.
+. Under *Compute Virtualization*, click *VMware Virtual Disk Development Kit (VDDK)*.
```

Comment:
In clicking that link, I didn't see "VMware Virtual Disk Development Kit (VDDK)". Should it be just "Virtual Disk Development Kit (VDDK)", which I do see?

Reply:
Good catch.